assets/quick-add.css

```diff
@@ -0,0 +1,213 @@
+.quick-add-modal {
```

Comment:
Anything we can do to ensure there's some margin below a scrollable modal? https://screenshot.click/15-40-ol71e-yb46p.mp4

Reply:
@kmeleta  thanks for noticing this! @LucasLacerdaUX similar to the mobile spacing, if possible let's make sure there is some pre-set margin below the modal at all times.